hedera-node/docs/design/services/token-service/airdrops/token-reject.md

```diff
@@ -140,10 +138,8 @@ All of the expected behaviour described below should be present only if the new
 - Given account with some NFT in its balance when `TokenReject` for the same NFT is performed then the `TokenReject` should succeed, the NFT should be transferred from the account to the token treasury and any other NFTs from the same collection should be left in the account
 - Given account with enough fungible tokens and NFTs in its balance when `TokenReject` for up to 10 transfers of fungible or NFTs is performed then the `TokenReject` should succeed and the whole amount of the fungible tokens, the specified NFTs from the account should should be transferred to the token treasury and any other NFTs from the same collection should be left in the account
 - Given account with some token in its balance and treasury account with `receiver_sig_required` enabled when `TokenReject` for the same token is performed then the `TokenReject` should succeed
-- Given account with some fungible token in its balance that is frozen when `TokenReject` for the same token is performed then the `TokenReject` should succeed
-- Given account with some fungible token in its balance that is paused when `TokenReject` for the same token is performed then the `TokenReject` should succeed
-- Given account with some NFT in its balance that is frozen when `TokenReject` for the same NFT is performed then the `TokenReject` should succeed and any other NFTs from the same collection should be left in the account
-- Given account with some NFT in its balance that is paused when `TokenReject` for the same NFT is performed then the `TokenReject` should succeed and any other NFTs from the same collection should be left in the account
+- Given account with some fungible token in its balance that is frozen when `TokenReject` for the same token is performed then the `TokenReject` should fail
+- Given account with some fungible token in its balance that is paused when `TokenReject` for the same token is performed then the `TokenReject` should fail
```

Comment:
Should we have a similar two lines for NFT tokens?

Reply:
Added.